docker/ts_config/timesketch.conf

```diff
@@ -0,0 +1,12 @@
+DEBUG = False
+SECRET_KEY = u'dockerdockerdocker'
```

Comment:
This needs to be generated later otherwise the installation is open for XSRF attackes etc. Maybe in the install script?